pico_limbo/src/handlers/play/set_player_position_and_rotation.rs

```diff
@@ -0,0 +1,33 @@
+use minecraft_packets::play::set_player_position_and_rotation::SetPlayerPositionAndRotationPacket;
+use minecraft_packets::play::synchronize_player_position_packet::SynchronizePlayerPositionPacket;
+use crate::server::client_state::ClientState;
+use crate::server::packet_handler::{PacketHandler, PacketHandlerError};
+use crate::server::packet_registry::PacketRegistry;
+use crate::server_state::ServerState;
+
+impl PacketHandler for SetPlayerPositionAndRotationPacket {
```

Comment:
Add tests for this handler and the handler for the `SetPlayerPositionPacket` as well for the following cases:
- Player is above `min_y_pos` → no packet should be sent
- Player is below `min_y_pos` → two packets should be sent

Run the tests with `cargo test`